src/desktop/components/main_layout/templates/scripts.jade

```diff
@@ -1,4 +1,4 @@
-- defaultOptions = {grecaptcha: true, stripev3: false, sailthru: true, marketo: true}
+- defaultOptions = {grecaptcha: true, marketo: true, onetrust: true, sailthru: true, stripev3: false}
```

Comment:
how do we over-ride these default options? via ENV?

Reply:
Not sure. @icirellik ?

Reply:
`options` are overwritten on a template-by-template basis, and merged in via [`Object.assign`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) below. 

If you search force for `-options` things should pop up:

<img width="396" alt="Screen Shot 2021-06-22 at 11 47 06 AM" src="https://user-images.githubusercontent.com/236943/122982374-a50e7f80-d34f-11eb-82e2-d965b457b61f.png">